class/defaults.yml

```diff
@@ -2,7 +2,10 @@ parameters:
   openshift4_console:
     namespace: openshift-console
     namespace_annotations:
-      openshift.io/node-selector: node-role.kubernetes.io/master=
+      openshift.io/node-selector: ''
```

Comment:
This change has been backported to 4.13, so we can update the config in the component unconditionally.